### ISSUE TITLE
Implement Flot Resize

### ISSUE THREAD

**Issue by rizkiandrianto** [rodrigowirth/react-flot#2]
You include jquery.flot.resize but it can't be used. Did you have any suggestion?

**Comment by rodrigowirth** [comment]
Hi @rizkiandrianto, can you post how you are creating the chart?

**Comment by rizkiandrianto** [comment]
Hi @rodrigowirth , here is my chart; 

The chart looks like only responsive when window is triggered to resize right?
Did you have any suggestion to resize when the parent is response to css animation?

```
import React, { PropTypes, Component } from 'react';
import ReactFlot from 'react-flot';

class SinChart extends Component{
  render () {
    var sin = [],
    cos = [];
    for (var i = 0; i < 14; i += 0.5) {
      sin.push([i, Math.sin(i)]);
      cos.push([i, Math.cos(i)]);
    }

    const data = [
    { data: sin, label: "clicks(x)"},
    { data: cos, label: "impression(x)"}
    ];

    const options = {
      series: {
        lines: {
          show: true
        },
        points: {
          show: true
        }
      },
      grid: {
        tickColor: "#e4e5e7",
        borderWidth: 1,
        borderColor: '#e4e5e7',
        color: '#6a6c6f'
      },
      colors: [ "#f15257", "#374552"],
    };

    return (
      <ReactFlot id={this.props.id} options={options} data={data} width={this.props.width || '100%'} height={this.props.height || "300px"} />
    )
  }
}

export default SinChart;

```

![anim](https://cloud.githubusercontent.com/assets/2457820/23658462/3c0546ee-0375-11e7-9d91-2f91434ac86e.gif)

**Comment by rodrigowirth** [comment]
Hi @rizkiandrianto,

Sorry for the late answer. Yes, it's based on window resize. We can find a way to trigger it based on the css. 

I'm on vacation right now. If you find a solution, could you please send a pull request? Or a comment on how to do?

Thank you very much,

**Comment by rodrigowirth** [comment]
Hi @rizkiandrianto, have you managed to make flot-resize works with react and webpack? I'm trying to solve this issue, but I'm dealing with an error related to `'requestAnimationFrame' of undefined`.

**Comment by rodrigowirth** [comment]
Hi @rizkiandrianto, I have just managed to apply the flot.resize plugin to react-flot. It's available in the new 1.1.0 version.

**Comment by rizkiandrianto** [comment]
Wow, I'll try it later... Sorry for late reply, on vacation for months...
Thank you @rodrigowirth ...
